openvpn/client_config.go

```diff
@@ -76,14 +74,14 @@ func newClientConfig(configDir string) *ClientConfig {
 
 // NewClientConfigFromSession creates client configuration structure for given VPNConfig, configuration dir to store serialized file args, and
 // configuration filename to store other args
-func NewClientConfigFromSession(vpnConfig *VPNConfig, configDir string, configFile string) (*ClientConfig, error) {
+func NewClientConfigFromSession(vpnConfig *VPNConfig, configDir string, runtimeDir string) (*ClientConfig, error) {
 
 	err := NewDefaultValidator().IsValid(vpnConfig)
 	if err != nil {
 		return nil, err
 	}
 
-	clientFileConfig := newClientConfig(configDir)
+	clientFileConfig := newClientConfig(runtimeDir, configDir)
```

Comment:
Arguments are swapped here

Reply:
not swapped. Wrong naming inside function. Fixed.